### PR TITLE
Fix #70 #93 remove json caching feature

### LINKED ISSUE #70: is json caching necessary?
See nsf/gocode#368.

You can complete unimported  packages now.

### PR DESCRIPTION
**Pull request by Shougo**
@zchee Please check the changes.

### CODE REVIEW

**Comment by zchee** [comment]
@Shougo Thanks, I'll check tomorrow.

File: README.md [hunk 1]
```diff
@@ -178,116 +176,6 @@ func NewFoo() *Foo {
...
-Recommend is `~/.cache/deoplete/go/$GOOS_$GOARCH` because this directly use other deoplete sources(e.g. deoplete-deji).
-
-Also, See [How to use static json caching](#how-to-use-static-json-caching)
-
-```vim
-let g:deoplete#sources#go#use_cache = 1
-let g:deoplete#sources#go#json_directory = '/path/to/data_dir'
-```
-
-### `g:deoplete#sources#go#cgo`
```
Comment (zchee) [inline README.md]: @Shougo It seems to work successfully. But why also delete `g:deoplete#sources#go#cgo` document?

**Review by zchee — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Shougo) [reply README.md]: Oh, it is my fault.

**Comment by Shougo** [comment]
I have fixed the documentation format.
Please check it.

**Comment by zchee** [comment]
@Shougo Thanks, I'll check later.

File: README.md [hunk 2]
```diff
@@ -8,29 +8,29 @@
 Go source for [deoplete.nvim](https://github.com/Shougo/deoplete.nvim) use [gocode](https://github.com/nsf/gocode).
 
 ## Overview
-Asynchronous Go completion for Neovim.
```
Comment (zchee) [inline README.md]: Ah, sorry but not nits, last two spaces need to for the new line.

Comment (zchee) [reply README.md]: It means, Do not trim the two spaces.

**Review by zchee — CHANGES_REQUESTED** [review]
@Shougo Sorry, PTAL.

File: README.md (hunk 2 above)
Comment (Shougo) [reply README.md]: Why?  I don't understand the two speces reason.

Comment (zchee) [reply README.md]: Oh, it's markdown(GitHub Flavor Markdown) specific. See below diff.
- https://github.com/zchee/deoplete-go/blob/master/README.md
- https://github.com/zchee/deoplete-go/blob/c1f372aa08f423ecb09d0a46726e96dde059674c/README.md

There is new line before the `Use,` word.

Comment (Shougo) [reply README.md]: Ah, I get it.

Comment (zchee) [reply README.md]: 🙇

**Comment by Shougo** [comment]
Please check it.

**Comment by zchee** [comment]
@Shougo LGTM. Merged.

**Comment by Shougo** [comment]
Thank you!

**Pull request merged**